assets/build/20_install.sh

```diff
@@ -76,6 +76,9 @@ case ${GITLAB_VERSION?} in
 11.4.*)
   git apply -v ${GITLAB_BUILD_DIR?}/patches/11.4/*
 ;;
+10.8.*)
```

Comment:
that order doesn't make sense, though it's just a cosmetic issue :)

Reply:
y tho